pandas/core/frame.py

```diff
@@ -2215,6 +2215,16 @@ def to_parquet(
            col1  col2
         0     1     3
         1     2     4
+
+        If you want get a buffer to the parquet content you can use a io.BytesIO object,
+        as long as you don't use partition_cols, which creates multiple files.
+
+        >>> import io
+        >>> f = io.BytesIO()
+        >>> df.to_parquet(f)
+        >>> f.seek(0)
+        0
+        >>> content=f.read()
```

Comment:
Nitpick but can you add whitespace around the = operator? I thought this would get picked up in our code checks..

Reply:
yes, sure.